core/templates/pages/exploration-editor-page/services/exploration-warnings.service.spec.ts

```diff
@@ -16,6 +16,12 @@
  * @fileoverview Unit tests for ExplorationWarningsService.
  */
 
+import {
+  HttpClient,
+  HttpXhrBackend,
+  // eslint-disable-next-line camelcase
```

Comment:
@vojtechjelinek, can you please take a look at this and tell me how can I disable this check? The linter issues error that `please do not disable camelcase and use single quotes instead if its property` but since this is not a property, I have to disable this check (just like UpgradedServices). This exact file is needed so that I can construct `ClassifierDataBackendApiService` object which is required for `StateClassifierMappingService` object which is required for these various test cases. Is there a way to disable errors raised by linter for disabling this checks?

Reply:
I think you will need to find the exact lint check which does it and then add an exception for this place.